Error compiling on Elixir 1.15.0/OTP 26.0.1 on M1 Mac with EVISION_PREFER_PRECOMPILED=true

## Environment
* Elixir: 1.15.0
* Erlang: 26.0.1
* Mac OS 12.3 (Monterey), M1 Pro
* EVISION_PREFER_PRECOMPILED=true (compiles without error when `false`)

### Current Behaviour
After installing Elixir 1.15.0-otp-26 and Erlang 26.0.1 the following compilation error occurs. I have tried:
* `mix deps.clean evision && deps.get` - same error
* `rm -rf ~/Library/Caches/evision*` - same error
```elixir
==> evision
CMake Warning:
  Ignoring empty string ("") provided on the command line.

Not searching for unused variables given on the command line.

-- The C compiler identification is AppleClang 13.1.6.13160021
-- The CXX compiler identification is AppleClang 13.1.6.13160021
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using PRIV_DIR: /Users/kip/Development/image/_build/dev/lib/evision/priv
-- Using ERTS_INCLUDE_DIR: /Users/kip/.asdf/installs/erlang/26.0.1/erts-14.0.1/include
CMake Error at CMakeLists.txt:27 (find_package):
  Could not find a package configuration file provided by "OpenCV" with any
  of the following names:

    OpenCVConfig.cmake
    opencv-config.cmake

  Add the installation prefix of "OpenCV" to CMAKE_PREFIX_PATH or set
  "OpenCV_DIR" to a directory containing one of the above files.  If "OpenCV"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!
See also "/Users/kip/Development/image/_build/dev/lib/evision/cmake_evision/CMakeFiles/CMakeOutput.log".
See also "/Users/kip/Development/image/_build/dev/lib/evision/cmake_evision/CMakeFiles/CMakeError.log".
incomplete build of OpenCV found in '/Users/kip/Development/image/_build/dev/lib/evision/cmake_opencv_4.7.0', please delete that directory and retry
make: *** [/Users/kip/Development/image/_build/dev/lib/evision/priv/evision.so] Error 1
could not compile dependency :evision, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile evision --force", update it with "mix deps.update evision" or clean it with "mix deps.clean evision"
```

@cocoa-xu I had tried that (since there is a helpful message to do so). Still no success:
```bash
% asdf current
elixir          1.15.0-otp-26   /Users/kip/.tool-versions
erlang          26.0.1          /Users/kip/.tool-versions
rust            stable          /Users/kip/.tool-versions

rm -rf /Users/kip/Development/image/_build/dev/lib/evision/cmake_opencv_4.7.0
% mix deps.clean evision
* Cleaning evision
% export EVISION_PREFER_PRECOMPILED=true
% mix deps.get
Resolving Hex dependencies...
Resolution completed in 0.071s
....
% iex -S mix
warning: Logger.warn/1 is deprecated. Use Logger.warning/2 instead
  /Users/kip/Development/image/deps/evision/mix.exs:272: Mix.Tasks.Compile.EvisionPrecompiled.warning_if_no_cacertfile!/1

==> evision
CMake Warning:
  Ignoring empty string ("") provided on the command line.

Not searching for unused variables given on the command line.

-- The C compiler identification is AppleClang 15.0.0.15000029
-- The CXX compiler identification is AppleClang 15.0.0.15000029
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using PRIV_DIR: /Users/kip/Development/image/_build/dev/lib/evision/priv
-- Using ERTS_INCLUDE_DIR: /Users/kip/.asdf/installs/erlang/26.0.1/erts-14.0.1/include
CMake Error at CMakeLists.txt:27 (find_package):
  Could not find a package configuration file provided by "OpenCV" with any
  of the following names:

    OpenCVConfig.cmake
    opencv-config.cmake

  Add the installation prefix of "OpenCV" to CMAKE_PREFIX_PATH or set
  "OpenCV_DIR" to a directory containing one of the above files.  If "OpenCV"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!
incomplete build of OpenCV found in '/Users/kip/Development/image/_build/dev/lib/evision/cmake_opencv_4.7.0', please delete that directory and retry
make: *** [/Users/kip/Development/image/_build/dev/lib/evision/priv/evision.so] Error 1
could not compile dependency :evision, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile evision --force", update it with "mix deps.update evision" or clean it with "mix deps.clean evision"
```

--- a/mix.exs
+++ b/mix.exs
@@ -755,7 +755,8 @@ defmodule Evision.MixProject do
             "CMAKE_OPTIONS" => cmake_options,
             "ENABLED_CV_MODULES" => enabled_modules,
             "TARGET_ABI" => System.get_env("TARGET_ABI", target_abi),
-            "EVISION_GENERATE_LANG" => System.get_env("EVISION_GENERATE_LANG", "elixir")
+            "EVISION_GENERATE_LANG" => System.get_env("EVISION_GENERATE_LANG", "elixir"),
+            "EVISION_PREFER_PRECOMPILED" => "false"
           }
 
           {[:elixir_make] ++ Mix.compilers(), make_env}

